### PR TITLE
docs: update Pulumi docs to reflect breaking changes

### PR DESCRIPTION
**Pull request by dirien**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.

## Updated docs pages

Please also include the path for the updated docs

- All Pulumi pages

### CODE REVIEW

File: docs/build-your-software-catalog/sync-data-to-catalog/iac/pulumi/pulumi.md [hunk 1]
```diff
@@ -180,26 +180,22 @@ The `properties` schema assigns a specified value to one of the entity's propert
...
-	],
-	# highlight-end
-	relations=[]
+  "myEntity",
+  identifier="myEntity",
+  title="My Entity",
+  blueprint="myBlueprint",
+  # highlight-start
+  properties=EntityPropertiesArgs(
+    string_props={
```
Comment (MPTG94) [inline docs/build-your-software-catalog/sync-data-to-catalog/iac/pulumi/pulumi.md:192]: Aside from string properties, all other property types are not updated to the latest syntax

File: docs/build-your-software-catalog/sync-data-to-catalog/iac/pulumi/examples/pulumi-managed-blueprint.md [hunk 2]
```diff
@@ -38,138 +37,91 @@ other = port.Blueprint(
...
+    calculation_properties={
+        "myCalculation": port.BlueprintCalculationPropertiesArgs(
+            title="My calculation property", calculation=".properties.myStringProp + .properties.myStringProp", type="string",
+        ),
+        "myCalculationWithMeta": port.BlueprintCalculationPropertiesArgs(
+            title="My calculation property with meta properties", calculation='.identifier + "-" + .title + "-" + .properties.myStringProp', type="string",
+        ),
+    },
+    relations={
+        "myRelation": port.BlueprintRelationsArgs(
```
Comment (MPTG94) [inline docs/build-your-software-catalog/sync-data-to-catalog/iac/pulumi/examples/pulumi-managed-blueprint.md:119]: The relation syntax is not fully up to date

**Review by MPTG94 — APPROVED** [review]
LGTM, left some comments for changes and updates that are remaining

File: docs/build-your-software-catalog/define-your-data-model/setup-blueprint/properties/string.md [hunk 3]
```diff
@@ -286,13 +286,13 @@ func main() {
...
 			Title:      pulumi.String("My Blueprint"),
       // highlight-start
-			Properties: port.BlueprintPropertyArray{
-				&port.BlueprintPropertyArgs{
-					Identifier: pulumi.String("myStringProp"),
-					Title:      pulumi.String("My String"),
-					Required:   pulumi.Bool(false),
-					Type:       pulumi.String("string"),
-				},
+			Properties: port.BlueprintPropertiesArgs{
```
[the remark this answers is not in the document]
Comment (dirien) [reply docs/build-your-software-catalog/define-your-data-model/setup-blueprint/properties/string.md:289]: Done

File: docs/build-your-software-catalog/define-your-data-model/setup-blueprint/properties/url.md [hunk 4]
```diff
@@ -255,17 +255,22 @@ const entity = new port.Blueprint("myBlueprint", {
...
   identifier: "myBlueprint",
   // highlight-start
-  properties: [
-    {
-      identifier: "myUrlProp",
-      title: "My url",
-      type: "string",
-      required: true,
-      format: "url",
+    properties: {
```
[the remark this answers is not in the document]
Comment (dirien) [reply docs/build-your-software-catalog/define-your-data-model/setup-blueprint/properties/url.md:258]: done

File: docs/build-your-software-catalog/sync-data-to-catalog/iac/pulumi/examples/pulumi-managed-blueprint.md (hunk 2 above)
Comment (dirien) [reply docs/build-your-software-catalog/sync-data-to-catalog/iac/pulumi/examples/pulumi-managed-blueprint.md:119]: Looks good for me and it deploy. What do you mean in detail

File: docs/build-your-software-catalog/sync-data-to-catalog/iac/pulumi/pulumi.md (hunk 1 above)
Comment (dirien) [reply docs/build-your-software-catalog/sync-data-to-catalog/iac/pulumi/pulumi.md:192]: done!

File: docs/build-your-software-catalog/sync-data-to-catalog/iac/pulumi/examples/pulumi-managed-blueprint.md (hunk 2 above)
Comment (danielsinai) [reply docs/build-your-software-catalog/sync-data-to-catalog/iac/pulumi/examples/pulumi-managed-blueprint.md:119]: Looking correct ✅

**Pull request merged**